example-app/android/build.gradle

```diff
@@ -8,9 +8,7 @@ buildscript {
         minSdkVersion = 21
         compileSdkVersion = Integer.parseInt(findProperty('android.compileSdkVersion') ?: "33")
         targetSdkVersion = Integer.parseInt(findProperty('android.targetSdkVersion') ?: "31")
-        if (hasProperty('android.kotlinVersion')) {
-            kotlinVersion = findProperty('android.kotlinVersion')
-        }
+        kotlinVersion = "1.8.21"
```

Comment:
Any particular reason for this removal?

Reply:
Otherwise I was getting the following error, but also I haven't fully figured out where this property is supposed to come from. Is there something else I can change to make it work? 

```
The Android Gradle plugin supports only Kotlin Gradle plugin version 1.5.20 and higher.
The following dependencies do not satisfy the required version:
project ':react-native-screens' -> org.jetbrains.kotlin:kotlin-gradle-plugin:1.4.10
```

Reply:
Is `android.kotlinVersion` maybe defined somewhere in the bitrise environment?

Reply:
Or maybe it's just a matter of not having a default, like the other properties above do. Something like:

```groovy
kotlinVersion = findProperty('android.kotlinVersion') ?: "1.8.21"
```

should work

Reply:
Updated for dev-app and example-app build.gradle files in https://github.com/stripe/stripe-terminal-react-native/pull/467/commits/8e1b245867637ea73181ccdfdf82150ae2f2cf98